cmd/network/verification.go

```diff
@@ -157,6 +160,7 @@ func (e *EgressVerification) Run(ctx context.Context) {
 
 		// Only suggest sending a service log if the failures are egress-url related
 		if !out.IsSuccessful() && len(out.GetEgressURLFailures()) > 0 {
+			fmt.Println("Consider increasing the egress timeout with the --egress-timeout option if you think the problem is timeout related.")
```

Comment:
This message feels problematic to me - when would an SRE running this be able to determine that increasing the timeout would be the appropriate option here vs sending the SL as-is?

I understand the reasoning behind adding the flag and I want to be clear that I understand why an SRE would want to use the flag - I'm wondering though if there's a better indicator of when this message may be more applicable, otherwise I see this specific message potentially causing confusion.

Reply:
Then, to make it clearer, maybe we should just run it with bigger timeout from beginning, which is probably not problematic, and then we do not need this message, which may be confusing ;)

Reply:
Hum, on a second thought, using a bigger default timeout may actually not be that a great idea.
Indeed as discussed with @bergmannf, we don't know what is the timeout used by pods / operators which are really targeting those endpoints.
Increasing the default timeout may actually leads to situations in which the `osdctl` command won't warn anymore while the pod / operator using the endpoint is actually timing out because it is using a different / lower timeout.
So finally I will simply remove the log as @iamkirkbater suggested and keep the default timeout to `2s`.